### ISSUE TITLE
Implement `Hasher` for `xxhash_rust::xxh32::Xxh32`

### ISSUE THREAD

**Issue by tgross35** [DoumanAsh/xxhash-rust#31]
It seems like Xxh32 does not implement `Hasher`

**Comment by DoumanAsh** [comment]
xxh32 produces 32bit hash and just cannot be used to implement `Hasher` which expects 64 bit output

If you want to use xxh32 then use its interface directly, although I do not recommend using xxh32 unless you're on resource constrained system

**Comment by tgross35** [comment]
I think that usually hashers are zero-extended or truncated to `u64` in order to meet the `Hasher` trait, e.g. `FxHash32` or `WyHash`.

I am not using it directly, just doing some benchmarking among different algorithms

**Comment by DoumanAsh** [comment]
Yes, but it serves no purpose other than being misleading. I only implement `Hasher` for algorithms that output `u64` nothing more nothing less (xxh3 algorithm can do u128 and u64 so `Hasher` there uses `u64` version)
If you're just going to use it one time, I suggest make your own wrapper that implement `Hasher` and use `Xxh32` methods directly
